cssdb.json

```
@@ -526,6 +526,15 @@
       }
     ]
   },
+  {
+    "id": "read-only-write",
```

This will be our first combo pseudo class. I like what you’ve done by dropping a word like `and`, but could we still include the `pseudo-class` suffix?

```suggestion
    "id": "read-only-write-pseudo-class",
```